src/app.js

```diff
@@ -1,21 +1,57 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs/promises');
+const { generateFilePath } = require('./helpers/generateFilePath');
+
+const PORT = process.env.PORT || 3003;
+
+const server = http.createServer(async(req, res) => {
+  if (req.url === '/favicon.ico') {
+    res.writeHead(204, { 'Content-Type': 'image/x-icon' });
+    res.end();
+
+    return;
+  }
+
+  const reqPath = req.url;
+
+  if (!reqPath.startsWith('/file')) {
+    res.setHeader('Content-Type', 'application/json');
+
+    const payload = {
+      message: 'The path should start with /file/',
+    };
+
+    res.end(JSON.stringify(payload, null, 2));
+
+    return;
+  }
+
+  const filePath = generateFilePath(reqPath);
+
+  try {
+    const file = await fs.readFile(filePath, 'utf8');
+
+    if (filePath.endsWith('.html')) {
+      res.setHeader('Content-Type', 'text/html');
+    } else if (filePath.endsWith('.css')) {
+      res.setHeader('Content-Type', 'text/css');
+    } else {
+      res.setHeader('Content-Type', 'text/plain');
+    }
```

Comment:
Would be better to use some strategy here instead of if statements. It much more readable and scalable